Library/Homebrew/tab.rb

```diff
@@ -226,6 +226,7 @@ def self.runtime_deps_hash(formula, deps)
       {
         "full_name"         => f.full_name,
         "version"           => f.version.to_s,
+        "revision"          => f.revision,
```

Comment:
```suggestion
        "version"           => f.pkg_version.to_s,
```

alternative approach

Reply:
Guess it depends on how to fix old bottles above.

Was initially also avoiding `PkgVersion.parse` as it's not 100% accurate in some cases but `any_installed_version` uses the same so is safe heree.